src/Components/Navbar/Navbar.jsx

```diff
@@ -1,5 +1,83 @@
+import React, { useState } from "react";
+// import { Link } from "react-router-dom";
+import { Icon } from "@iconify-icon/react";
+import styles from "./Navbar.module.scss";
+import logo from "./GDSC-Logo.png";
+
 const Navbar = () => {
-  return <nav>Navbar</nav>;
+  const [isActiveHome, setisActiveHome] = useState(true);
+  const [isActiveResources, setisActiveResources] = useState(false);
+
+  const handleClickHome = () => {
+    setisActiveHome(true);
+    setisActiveResources(false);
+  };
+
+  const handleClickResources = () => {
+    setisActiveResources(true);
+    setisActiveHome(false);
+  };
+
+  const [toggle, setToggle] = useState(false);
+
+  const handleToggle = () => {
+    setToggle(!toggle);
+  };
+
+  const changeToggle = () => {
+    if (window.scrollY >= 90 && window.innerWidth > 1000) {
+      setToggle(true);
+    } else {
+      setToggle(false);
+    }
+  };
+  window.addEventListener("scroll", changeToggle);
```

Comment:
Checkout better implementation of toggling open and close Navbar. Google more alternatives in React